### ISSUE TITLE
A simple toggle for permanent private browsing

### ISSUE THREAD

**Issue by Pen15Injector** [futo-org/grayjay-android#1505]
### Use Cases

Maybe if you have ptsd about your dad finding porn in your search history, and many others.

### Proposal

One simple permanent toggle for:
No search history
No watch history
No watch time tracking

If you choose to subscribe to things your subscriptions are still saved

**Comment by ismacrazy7** [comment]
Good idea, I'm tired of activating Private Mode every time I open the App.
